tests/v2/actions/provisioning/creates.go

```diff
@@ -66,10 +66,6 @@ const (
 	rke1NodeCorralName                   = "rke1registerNode"
 )
 
-var (
-	updateConfig = true
```

Comment:
I have one question, why was it removed?

Reply:
May be needs a rebase

Reply:
good catch, pushed the change just now.